sky/utils/controller_utils.py

```diff
@@ -517,6 +517,30 @@ def get_controller_resources(
         if custom_controller_resources_config is not None:
             controller_resources_config_copied.update(
                 custom_controller_resources_config)
+        # Compatibility with the old way of specifying the controller autostop
```

Comment:
since this only happens for initial retrieval of the controller resources, will there be an backward compat issue for controllers started before this PR?

Reply:
I don't think so. What issue did you have in mind?